tox.bat

```diff
@@ -0,0 +1,2 @@
+@if not defined BASEPYTHON set BASEPYTHON=python
+@tox.exe %*

```

Comment:
On Windows only, running `tox` (with or without arguments) will default to this script, which ensures that `BASEPYTHON` is set (using the `python` executable of the venv, or the one in `PATH` if not using venvs for some reason)